apps/verticalhome/js/app.js

```diff
@@ -86,6 +88,16 @@
       }
     },
 
+    isinPinnedApp: function(result) {
+      var pinnedItems = app.pinnedAppsManager.items;
+      for (var i=0; i<pinnedItems.length; i++){
+        if(pinnedItems[i].entry!=null &&
+          (result.app.manifestURL == pinnedItems[i].entry.manifestURL)){
```

Comment:
May be it would be greater make a new variable for example
var resManifest = result.app.manifestURL;
for (var i=0; i<pinnedItems.length; i++){
       if (!pinnedItems[i].entry && resManifest == pinnedItems[i].entry.manifestURL){

Or if you get lint errors you can also pinnedItems[i].entry keep in new variable too.